src/components/breadcrumb/LinkingBreadcrumb/LinkingBreadcrumb.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { BaseBreadcrumbProps, Breadcrumb } from '../Breadcrumb/Breadcrumb'
+import { BreadcrumbLink } from '../BreadcrumbLink/BreadcrumbLink'
+
+interface LinkingBreadcrumbProps extends BaseBreadcrumbProps {
+  href: string
+  anchorProps?: JSX.IntrinsicElements['a']
+  spanProps?: JSX.IntrinsicElements['span']
```

Comment:
Maybe redundant prop? Not used in component

Reply:
Yea per your advice and @suzubara I'm killing the LinkingBreadcrumb. It felt like a nice candidate for a compositional convenience component for developers who wanted default functionality with low complexity. At the end of the day it doesn't really fit with the convention of this library, and ultimately adds a maintenance burden to maintainers of react-uswds.

Reply:
Sounds good.  And i hear you. Its always a bit weird (for me personally) to juggle the difference between the library and the code we write day to day.